hackathon_site/dashboard/frontend/src/api/api.ts

```diff
@@ -50,6 +50,25 @@ export const stripHostname = (url: string): string => {
     return parsed.pathname + parsed.search + parsed.hash;
 };
 
+export const stripHostnameReturnFilters = (
+    url: string | null
+): { path: string; filters?: { [key: string]: any } } => {
+    if (url) {
+        const parsed = new URL(url);
+        const filters: { [key: string]: any } = {};
+        if (parsed.search && filters) {
```

Comment:
The `&& filters` here isn't doing anything, since the truthyness of an empty object in Javascript is true for some reason

Reply:
but filters is an optional argument

Reply:
D'oh, good point :man_facepalming: